### PR TITLE
feat(unit-testing): added coverage collection and printing joined report

### PR DESCRIPTION
**Pull request by pawelleszczewicz**
added stuff created by Alex, added also html report

### CODE REVIEW

**Comment by aifrim** [comment]
> added stuff created by Alex, added also html report

you stole my work!

**Comment by pawelleszczewicz** [comment]
if i would steal it i would not mention your name :P

File: jest.config.js [hunk 1]
```diff
@@ -1,10 +1,38 @@
+const esModules = ['lodash-es', '@ngrx'].join('|');
```
Comment (aifrim) [inline jest.config.js]: we don't have any `esModules` to ignore :)

Comment (pawelleszczewicz) [reply jest.config.js]: done

**Pull request closed without merging**